libs/design/src/molecules/list/list-item/list-item.component.ts

```diff
@@ -15,4 +15,22 @@ export class DaffListItemComponent {
 
   @ContentChild(DaffPrefixDirective, { static: false }) _prefix: DaffPrefixDirective;
   @ContentChild(DaffSuffixDirective, { static: false }) _suffix: DaffSuffixDirective;
+
+  constructor(private elementRef: ElementRef) {}
+
+  /**
+   * Sets the role for a regular `<daff-list-item>` to listitem.
+   */
+  @HostBinding('attr.role') get role() {
+    return this._isAnchor() ? null : 'listitem';
```

Comment:
Needs test coverage.

Reply:
Updated.

Reply:
What about the other side of the conditional?

Reply:
Updated.